<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1183</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Altas Flight</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>730</x>
         <y>0</y>
         <width>324</width>
         <height>711</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <widget class="qfi_ASI" name="graphicsASI">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>300</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>300</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::NoFrame</enum>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
          </property>
          <property name="interactive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="qfi_EADI" name="graphicsEADI">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>300</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>300</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::NoFrame</enum>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
          </property>
          <property name="interactive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>703</x>
         <y>-40</y>
         <width>20</width>
         <height>771</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>720</x>
         <y>335</y>
         <width>451</width>
         <height>41</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>671</width>
         <height>691</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Source Code Pro Medium</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Setting</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="2">
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="I_Value">
              <property name="font">
               <font>
                <family>Source Code Pro</family>
                <pointsize>11</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QDoubleSpinBox" name="D_Value">
              <property name="font">
               <font>
                <family>Source Code Pro</family>
                <pointsize>11</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <family>Source Code Pro</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>D:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>Source Code Pro</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>P:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="P_Value">
              <property name="font">
               <font>
                <family>Source Code Pro</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="minimum">
               <double>0.000000000000000</double>
              </property>
              <property name="maximum">
               <double>10099999.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>Source Code Pro</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>I:</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="1">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>582</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="2">
          <widget class="QWidget" name="widget_4" native="true">
           <widget class="QLabel" name="label_6">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>110</y>
              <width>49</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label_7">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>10</y>
              <width>121</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>PID Tuning:</string>
            </property>
           </widget>
           <widget class="QTextEdit" name="textEdit">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>40</y>
              <width>331</width>
              <height>491</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>Source Code Pro</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
            </property>
            <property name="documentTitle">
             <string/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::TextInteractionFlag::NoTextInteraction</set>
            </property>
           </widget>
          </widget>
         </item>
         <item row="1" column="3">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>582</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="2">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>408</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>Servo</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Remote Controll</string>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qfi_ASI</class>
   <extends>QGraphicsView</extends>
   <header>qfi/qfi_ASI.h</header>
  </customwidget>
  <customwidget>
   <class>qfi_EADI</class>
   <extends>QGraphicsView</extends>
   <header>qfi/qfi_EADI.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
